DESCRIPTION.md

```diff
@@ -0,0 +1,734 @@
+# Introduction
+
+Welcome to bunq!
+
+- The bunq API is organised around REST. JSON will be returned in almost all responses from the API, including errors but excluding binary (image) files.
+- All calls made through bunq Doc are executed on a sandbox environment. No real money is used and no transactions to external bank accounts can be done.
+- Please configure your implementation to send it's API requests to https://sandbox.public.api.bunq.com/v1.
+- There is an Android client available of our app that connects to the bunq Sandbox environment.
+- If you have a developer key, please proceed to the Settings page and enter this key to enable execution of test calls.
+
+## Get Started
+
+1. Create a user account with your phone. Afterwards, you can use this account to create an API key from which you can make API calls. You can find API key management under 'Profile' -\> 'Security'.
```

Comment:
We have Tinker for this now?

Reply:
Nein. Tinker is for playtime not actually implementation. Then you should indeed go and manually create the key 😊